local-setup/github-action-config.json

```diff
@@ -12,7 +12,9 @@
       "source": "bin",
       "flags": [
         "-P",
-        "2000"
+        "2000",
+        "-U",
```

Comment:
Untrusted ws server on worker side can now be set explicitly at worker start up.